cmd/agent/install_mac_os.sh

```diff
@@ -202,9 +202,17 @@ cmd_agent="$cmd_real_user /opt/datadog-agent/bin/agent/agent"
 
 cmd_launchctl="$cmd_real_user launchctl"
 
-function new_config() {
+function sed_inplace_arg() {
     # Check for vanilla OS X sed or GNU sed
-    i_cmd="-i ''"
+    if [ "$(sed --version 2>/dev/null | grep -c "GNU")" -ne 0 ]; then
+        echo "-i"
+    fi
+
+    echo "-i ''"
+}
+
+function new_config() {
+    i_cmd="$(sed_inplace_arg)"
     if [ "$(sed --version 2>/dev/null | grep -c "GNU")" -ne 0 ]; then i_cmd="-i"; fi
```

Comment:
seems like this line can go now

Reply:
You're completely right, I forgot that :facepalm: Thanks!

Reply:
thanks for being so responsive! this has been fun!